### PR TITLE
REDDEV-757 Pass Security Scan

### PR DESCRIPTION
**Pull request by saligiad**
Made changes enabling EM to pass the REDCap Security Scanner, and to maintain consistency

* Applied `Octri\Vizr` namespace to PHP files (and template file) producing errors
* Applied `Octri\Vizr` namespace to remaining PHP files for consistency 
* Ensure references to non-module entities fallback to global namespace

### CODE REVIEW

File: lib/persist.php [hunk 1]
```diff
@@ -107,6 +108,6 @@ function sanitizeChartConfig($charts) {
   array("data" => $data),
   array("response" => $response)
 ));
-REDCap::logEvent("Vizr chart configuration changed", $logData, NULL/*sql*/,
+\REDCap::logEvent("Vizr chart configuration changed", $logData, NULL/*sql*/,
```
Comment (heathharrelson) [inline lib/persist.php:111]: I wonder if we should use [module logging](https://github.com/vanderbilt-redcap/external-module-framework-docs/blob/main/methods/logs.md), now that it's availble.

Comment (saligiad) [reply lib/persist.php:111]: So something like?
```
- \REDCap::logEvent("Vizr chart configuration changed", $logData, NULL/*sql*/,
+ $module->log("Vizr chart configuration changed", 
+     [
+         "config_data" => $data, 
+         "response" => $response,
+         "PID" => $project_id
+     ]
+ );
```

Comment (heathharrelson) [reply lib/persist.php:111]: That looks reasonable, but I've never used it before. I'd suggest trying different things locally and seeing what shows up in the external module logs interface. The link is in the Control Center, next to the link to manage EMs.

**Review by aeyates — APPROVED** [review]

**Review by heathharrelson — APPROVED** [review]
Thanks for taking this on. Looking at the [API documentation](https://github.com/vanderbilt-redcap/external-module-framework-docs/blob/main/methods/README.md), it seems like the `$module` object has gained a lot of functionality in the years since we implemented Vizr. After merging this, it might be worth another pass through the module to replace use of `\REDCap` methods with their `$module` equivalents where possible.

**Pull request merged**